rotkehlchen/tests/unit/decoders/test_hop.py

```diff
@@ -85,3 +85,35 @@ def test_hop_optimism_eth_receive(database, optimism_inquirer):
             extra_data={'sent_amount': '0.2'},
         )]
     assert expected_events == events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('optimism_accounts', [['0x4bBa290826C253BD854121346c370a9886d1bC26']])
+def test_hop_optimism_eth_receive_2(database, optimism_inquirer, optimism_accounts):
```

Comment:
Also describe in the docstring what's the difference with the previous one.

```suggestion
def test_hop_optimism_eth_receive_no_event(database, optimism_inquirer, optimism_accounts):
```


Can you tell me also what did you do differently? Why is the `TransferFromL1Completed` event missing?